### PR TITLE
Allow to prepend bricks

### PR DESCRIPTION
**Pull request by Yuyuu**
Hello,
I'm making this PR to provide a way to prepend bricks.
Apparently this was started with #53 but not merged.

### CODE REVIEW

File: src/angular-masonry.js [hunk 1]
```diff
@@ -52,7 +52,7 @@
         $element.addClass('loaded');
       }
 
-      this.appendBrick = function appendBrick(element, id) {
+      this.appendBrick = function appendBrick(element, id, prepend) {
```
Comment (passy) [inline src/angular-masonry.js]: I'm not a fan of boolean arguments. Mind splitting this up and turning this into a `prependBrick` function instead?

**Comment by passy** [comment]
Thanks for the update!

**Comment by Yuyuu** [comment]
What do you think of renaming appendBrick to addBrick or attachBrick and passing the method ('appended' | 'prepended') instead of the boolean?
I think it can be a good way to avoid duplication.

**Comment by passy** [comment]
@Yuyuu Sorry for the delay. Yes, that looks way better! :cake: Would you mind also adding a paragraph to the readme about the new feature? :)

Happy to merge afterwards.

**Comment by Yuyuu** [comment]
Added it in the Attributes section. Hope this is what you had in mind.

**Comment by passy** [comment]
Brilliant! Yes, that's perfect. Thanks for the contribution! :snowman:

**Pull request merged**